packages/m2m-gateway/src/services/purposeTemplateService.ts

```diff
@@ -324,5 +324,31 @@ export function purposeTemplateServiceBuilder(
 
       return toM2MGatewayApiPurposeTemplate(data);
     },
+    async linkEServicesToPurposeTemplate(
+      purposeTemplateId: PurposeTemplateId,
+      eserviceIds: string[],
+      { headers, logger }: WithLogger<M2MGatewayAppContext>
+    ): Promise<m2mGatewayApi.EServiceDescriptorPurposeTemplate[]> {
```

Comment:
After updating to 204 no more need to return a value here
```suggestion
    ): Promise<void> {
```